parking-spot-submodule/src/main/java/ua/lviv/iot/application/service/ParkingSpotServiceInPort.kt

```diff
@@ -0,0 +1,21 @@
+package ua.lviv.iot.application.service
+
+import com.mongodb.client.result.DeleteResult
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+import ua.lviv.iot.domain.ParkingSpot
+
+interface ParkingSpotServiceInPort {
```

Comment:
```suggestion
interface ParkingSpotInPort {
```
InPort is substitution for Service

Reply:
✅